buildlib/tools/common.sh

```diff
@@ -70,6 +70,10 @@ make_clean() {
 	$MAKEP ${1:-clean}
 }
 
+has_doca() {
+    rpm -qa 'doca-sdk-gpunetio-devel*' 2>/dev/null | grep -q .;
```

Comment:
1. why needed, doesn't ucx find it automatically?
2. can we check the path /opt/mellanox/doca for gpunetio header files?

Reply:
1. why not? what is the issue? can we fix it in ucx autoconf? so it would work OOB for users?